addons/repair/models/stock_move.py

```diff
@@ -72,13 +72,15 @@ def create(self, vals_list):
             move.group_id = move.repair_id.procurement_group_id.id
             move.origin = move.name
             move.picking_type_id = move.repair_id.picking_type_id.id
-            repair_moves |= move
 
             if move.state == 'draft' and move.repair_id.state in ('confirmed', 'under_repair'):
                 move._check_company()
                 move._adjust_procure_method()
-                move._action_confirm()
-                move._trigger_scheduler()
+                exploded = move._action_confirm()
```

Comment:
(Nitpicking)
In a lot of cases, the method will return the same move. So let's make the name simpler
```suggestion
                moves = move._action_confirm()
```